migrations/20230328-initdb-stageBuilds.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}stageBuilds`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.createTable(
+                table,
+                {
+                    id: {
+                        allowNull: false,
+                        autoIncrement: true,
+                        primaryKey: true,
+                        type: Sequelize.INTEGER.UNSIGNED
+                    },
+                    stageId: {
+                        type: Sequelize.DOUBLE
+                    },
+                    status: {
+                        type: Sequelize.TEXT
+                    },
+                    groupEventId: {
```

Comment:
I believe we will have one snapshot for each event
```suggestion
                    eventId: {
```